src/kOS/Screen/TermWindow.cs

```diff
@@ -108,6 +131,39 @@ public void Awake()
             DontDestroyOnLoad(gObj);
             popupEditor = (KOSTextEditPopup)gObj.GetComponent(typeof(KOSTextEditPopup));
             popupEditor.SetUniqueId(UniqueId + 5);
+            
+            customSkin = BuildPanelSkin();
+
+            GameEvents.onHideUI.Add (OnHideUI);
+			GameEvents.onShowUI.Add (OnShowUI);
```

Comment:
Odd.  I have my editor configured to indent with only spaces.  Maybe these were tabs I actually typed in.

ohhhh... I cut-n-pasted from a code example I saw in the forums.  I bet the tabs were in the pasted text.